### ISSUE TITLE
Allow Binders to list Curated Lists

### ISSUE THREAD

**Issue by m-aiello** [massgov/feature-requests#127]
We have many instances where we want to group curated lists together into a binder format, but binders do not currently support curated lists.

Examples include grouping lists of circular letters. Here is a particularly large compilation of circular letter pages, which we currently formatted as a curated list that branches into more curated lists: https://www.mass.gov/lists/circular-letters-health-care-facility-licensure-and-certification

If this was a binder format, people could easily see a heading up top to get them back up the chain to other similar circular letter pages. Similarly, we would need a binder heading of "Circular letters."

**Comment by m-aiello** [comment]
I wanted to share another high-level situation we've encountered.

We currently have [this page for listing all the proposed amendments to regulations in DPH](https://edit.mass.gov/service-details/proposed-amendments-to-regulations-department-of-public-health). All the subpages are curated lists. As such, they are not linked smartly and have no parent-child relationships.

Additionally, we've found that some of the sub-pages should actually be broken up into multiple pages. [This page is an emaple](https://edit.mass.gov/lists/proposed-amendments-to-regulations-board-of-registration-in-nursing). While the analytics show that there are no comments on the above page, our liaison says he personally receives feedback from internal and external sources, including news sources that report on these, that the page is confusing. This is mostly due to how there are multiple proposed amendments on the same page, rather than dedicated pages for each one.

We would like to make dedicated curated list pages for each proposed amendment, rather than lump them together. Of course, doing so in the current system means another service-detail page above that with links to the dedicated curated lists.

This is another situation where the binder template would help us immensely in smartly grouping and properly linking this content. If binders could display curated lists, we could turn the general amendments page into a binder, and when there are more than one proposed amendments for a specific program, like nursing, we can create individual curated lists and "group" them together on the binder.

More updates to these pages are on their way, and it would be so helpful for everyone to have a binder smartly link to curated lists.

Thank you for listening,
Michael
